src/modules/access/access.service.ts

```
@@ -128,250 +60,141 @@ type Role = 'admin' | 'underboss' | 'caporegime' | 'host' | null;
+   * @returns {Promise<AccessResult>} Detailed access information including cities, regions, and countries
    */
-  async getAccessRole(telegram_id: string): Promise<Role> {
+  async getUserAccess(telegram_id: string): Promise<AccessResult | null> {
     const adminIds: string[] = process.env.ADMIN_IDS
       ? process.env.ADMIN_IDS.split(',').map((id) => id.trim())
       : [];
 
     if (adminIds.includes(telegram_id)) {
-      return 'admin';
-    }
+        const allRegions: RegionData[] = await this.knexService
```

ESLint

[error] Delete `··`

(prettier/prettier)